test/filterTracking.test.js

```diff
@@ -0,0 +1,80 @@
+import {buildFilterTrackingObj} from '../src/modules/filterTracking/buildFilterTrackingObj';
+import {cleanLabelName} from '../src/modules/filterTracking/cleanLabelName';
+import {checkFilters} from '../src/modules/filterTracking/checkFilters';
+
+document.body.innerHTML =
+    '<li id="subjects">'+
```

Comment:
Most of the DOM is not being used in the test. So my advice would be to just use the nodes that are required for the test.

Reply:
Done